auth-foundation/src/main/java/com/okta/authfoundation/client/OidcConfiguration.kt

```diff
@@ -74,10 +73,6 @@ class OidcConfiguration private constructor(
     /** The factory for creating a new instance of [Cache]. [Cache] is used to optimize network calls by the SDK. */
     @property:InternalAuthFoundationApi
     val cacheFactory: suspend () -> Cache = AuthFoundationDefaults.cacheFactory,
-
-    /** The CookieJar used for the network calls used by the SDK. */
-    @property:InternalAuthFoundationApi
-    val cookieJar: CookieJar = AuthFoundationDefaults.cookieJar
```

Comment:
Removing this property access here allows us to set cookieJar after OidcConfiguration has been created (and before any network calls are made). This is useful mostly for IDX